docs/issues/Issues.md

```diff
@@ -0,0 +1,37 @@
+#Issues
+
+##Labels
+All labels available: https://github.com/HeRAMS-WHO/herams-backend/labels
+
+##Creating an issue
+###Description
+When creating an issue be descriptive as possible. A good approach is:
+
+"I am logged in as `...` with `...`, and am on page `...`. I want to `...`"
```

Comment:
We should use issue templates instead of relying on the user.

Reply:
Didn't know Github had such things but would defenitely be the next step